testing/engine/operators_with_files.go

```diff
@@ -34,11 +34,26 @@ var _ = profile.RegisterProfile(profile.Profile{
 				},
 			},
 		},
+		{
+			Title: "operator of ip match from file",
+			Stages: []profile.Stage{
+				{
+					Stage: profile.SubStage{
+						Output: profile.ExpectedOutput{
+							TriggeredRules:    []int{20},
+							NonTriggeredRules: []int{21},
+						},
+					},
+				},
+			},
+		},
 	},
 	Rules: `
 SecRule ARGS_NAMES "@pmFromFile pmFromFile-01.dat" "id:1,log"
 SecRule REQUEST_COOKIES:def "@pmFromFile pmFromFile-01.dat" "id:3,log"
 SecRule REQUEST_COOKIES_NAMES "@pmFromFile pmFromFile-01.dat" "id:5,log"
 SecRule REQUEST_HEADERS_NAMES "@pmFromFile pmFromFile-01.dat" "id:10,log"
+SecRule REMOTE_ADDR "@ipMatchF netranges-01.dat" "id:20,log"
```

Comment:
I see that you are adding a couple of rules to be tested with `@ipMatchF`. Could you also add the same lines for `@ipMatchFromFile`? Just like we have now the same matching for `@pmFromFile` and `@pmf`